cypress/pom/MAT/WI/MeasureComposer.js

```diff
@@ -181,6 +182,8 @@ export const populationsListItems = '#LeftPackagePanel .GB-MJYKBEB > Div > [__id
 export const populationListItemsSecondary = '#LeftPackagePanel > [style="position: relative; zoom: 1;"] > .GB-MJYKBEB > :nth-child(1) > .GB-MJYKBAB > div'
 export const addAllItemsToGrouping = '#AddAllClauseToRight'
 export const saveGrouping = '#Save_Grouping_button'
+export const measurePackagerWarning1 = '.alert-danger > :nth-child(1)'
+export const measurePackagerWarning2 = '.alert-danger > :nth-child(2)'
```

Comment:
These are duplicates of 
export const groupingErrorOne = '.alert-danger > :nth-child(1) > b'
export const groupingErrorTwo = '.alert-danger > :nth-child(2) > b'

Lets try to use these